### PR TITLE
Close #320

### LINKED ISSUE #320: Duplicate campaigns
Some campaigns are appearing more than once in the Campaigns panel

### PR DESCRIPTION
**Pull request by kamicut**
- I removed the duplicates manually from staging
- I added a constraint to the database

### CODE REVIEW

File: api/src/services/tm_types/tm2.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ class TM2API {
         // not found
         return db('campaigns').insert(obj)
       } else {
-        return db('campaigns').where('tm_id', obj.tm_id).update(obj)
+        return db('campaigns').where('id', rows[0].id).update(obj)
```
Comment (alyssadelaine) [inline api/src/services/tm_types/tm2.js]: @kamicut Why is this necessary if the column is constrained to be unique?

Comment (kamicut) [reply api/src/services/tm_types/tm2.js]: Because we might have multiple tasking managers, so we select the project depending on tasker_id _and_ tm_id

Comment (alyssadelaine) [reply api/src/services/tm_types/tm2.js]: @kamicut My comment here was in reference to `rows[0]`, since there should only be one item returned if the column has a uniqueness constraint, right?

Comment (kamicut) [reply api/src/services/tm_types/tm2.js]: ah, because knex always returns an array even for a single row, so it's sure to be the first item in that array.
